packages/rainbowkit/src/components/ChainModal/ChainModal.test.tsx

```diff
@@ -1,21 +1,27 @@
 import user from '@testing-library/user-event';
 import React from 'react';
 import { describe, expect, it } from 'vitest';
-import { arbitrum, goerli, mainnet, optimism } from 'wagmi/chains';
+import {
+  arbitrum,
+  avalanche,
+  fantom,
+  mainnet,
+  optimism,
+  polygon,
+} from 'wagmi/chains';
 import { renderWithProviders } from '../../../test/';
-import { Chain } from '../RainbowKitProvider/RainbowKitChainContext';
 import { ChainModal } from './ChainModal';
 
 describe('<ChainModal />', () => {
   it('Unsupported chain', async () => {
     const { findByText } = renderWithProviders(
-      <ChainModal onClose={() => {}} open />,
+      <ChainModal mockChainId={1} onClose={() => {}} open />,
       {
-        chains: [mainnet], // only supports mainnet
+        chains: [polygon],
         mock: true,
-        mockOptions: { chainId: goerli.id }, // is connected to goerli
       },
     );
+
```

Comment:
wagmi beta doesn't have `chainId` when you do [account mocks](https://beta.wagmi.sh/react/api/connectors/mock#mock). I made sure to pass in a prop called `mockChainId` for now just to mock things and make sure behaviours work as expected.

Probably can rename this `initialChainId`, but seems to work for now.